contrib/setup-cms.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+dep_dir="deps"
+install_dir=$(readlink -fm ${dep_dir}/install)
+
+[ ! -d "${install_dir}" ] && mkdir -p "${install_dir}"
+
+dep="cms"
+
+cd ${dep_dir}
+git clone https://github.com/msoos/cryptominisat ${dep}
+cd ${dep}
+mkdir build && cd build
+cmake -G Ninja -DCMAKE_INSTALL_PREFIX:PATH=${install_dir} ..
```

Comment:
Do we need to use the Ninja generator? It adds a prereq we don't need.

Reply:
So, I actually removed Ninja *from the contrib scripts* as I agree that not everyone will have it.

I **have not** removed it from the GitHub Action.